### DISCUSSION TITLE
GitHub wiki link

### DISCUSSION THREAD

**Discussion by VPKSoft** [mkdocs/mkdocs#2371] [Q&A] [answered]
Hi,
I'm trying to create static HTML help from GitHub wiki markdown. How can I make the links to point at HTML pages?
I.e. markdown link definition `[API definition](API-defintion)` would point to `API-defintion.html` instead of `API definition` which is an "invalid" link.

**Comment by waylan** [comment, marked as answer]
How to link to pages is documented [here][1]. Note that the links must point to the source file (end in `.md`). 

Of course, this is not compatible with GitHub's wiki. MkDocs uses standard Markdown for links. GitHub's wiki does not. If you would like to support the wiki syntax, you may need to use (create) a custom Markdown extension.

[1]: https://www.mkdocs.org/user-guide/writing-your-docs/#internal-links

**Reply by Andre601** [reply]
There is a plugin that adds `[[text|link]]` syntax.

**Reply by VPKSoft** [reply]
Do you have an URL for the plugin?

**Reply by VPKSoft** [reply]
Found this: [EZLinks](https://github.com/orbikm/mkdocs-ezlinks-plugin). Seems to do the job with the following setting:
```yml
plugins:
  - ezlinks 
```
